release.yaml

```
@@ -1,22 +1,25 @@
-version: '1.0.6'
+version: '1.0.7'
 changelog: |-
-  New features and improvements
+  :tada: New features and improvements
```

```suggestion
  New features and improvements
```
Really need to fix that emoji parser